.msync.yml

```diff
@@ -1 +1 @@
-modulesync_config_version: '0.21.0'
+modulesync_config_version: '1.2.1'
```

Comment:
in the future please don't sync from HEAD, checkout the latest tag and use that instead